src/bci_build/package/micro/README.md.j2

```diff
@@ -2,12 +2,36 @@
 {% include 'badges.j2' %}
 
 ## Description
-This image is similar to Minimal but without the RPM package manager.
+The `bci-micro` image is even smaller than `bci-minimal` because it also
+does not include the RPM package manager.
+
 The primary use case for the image is deploying static binaries produced
-externally or during multi-stage builds. As there is no straightforward
-way to install additional dependencies inside the container image,
-we recommend deploying a project using the Minimal image only
-when the final build artifact bundles all dependencies and has no
-external runtime requirements (like Python or Ruby).
+externally or during multi-stage builds.
+
+
+## Usage
+
+As there is no straightforward way to install additional
+dependencies inside the container image, we recommend deploying a project
+using the Minimal image only when the final build artifact bundles all
+dependencies and has no external runtime requirements.
```

Comment:
If there are no external dependencies, what prevents it from being Scratch?
What advantages do I get from bci-micro?
SCC subscription without zypper?
A barebones image with minimal packages from SLE?

Reply:
the latter. I've reworded it to be clearer.